openwisp_radius/tasks.py

```diff
@@ -231,3 +233,37 @@ def get_radius_attributes(user):
                 f' RadiusAccounting object of "{user}" user'
             )
     RadiusAccounting.objects.bulk_update(updated_sessions, fields=["groupname"])
+
+
+@shared_task
+def process_radius_batch(batch_id, number_of_users=None):
+    RadiusBatch = load_model("RadiusBatch")
+    channel_layer = get_channel_layer()
+    group_name = f"radius_batch_{batch_id}"
+    batch = None
+
+    try:
+        batch = RadiusBatch.objects.get(pk=batch_id)
```

Comment:
Please Use the same pattern we use in other modules, eg:

https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/tasks.py#L23-L29
https://github.com/openwisp/openwisp-controller/blob/master/openwisp_controller/config/tasks.py#L43-L49